doc/api/debugger.md

```diff
@@ -202,3 +202,23 @@ To start debugging, open the following URL in Chrome:
 ```
 
 [TCP-based protocol]: https://github.com/v8/v8/wiki/Debugging-Protocol
+
+## Trace Event Support for Node.js
+
+Trace Event provides a mechanism to centralize tracing information generated by
+V8, Node core, and userspace code.
+
+Tracing can be enabled by passing the `--enable-tracing` flag when starting a
+Node.js application.
+
+The set of categories for which traces are recorded can be specified using the
+`--enabled-categories` flag followed by a list of comma separated category names.
+By default the `node` and `v8` categories are enabled.
+
+```txt
+node --enable-tracing --enabled-categories=v8,node server.js
+```
+
+Running Node.js with tracing enabled will produce log files that can be opened
+in the [`chrome://tracing`](https://www.chromium.org/developers/how-tos/trace-event-profiling-tool)
+tab of Chrome.
```

Comment:
Is there more info somewhere on this format, We should have some kind of tool available to allow the tracefile to be converted to a human readable format without requiring chrome, ideally something bundled with Node.js itself.  And then leave more sophisticated visualization/searching etc to external tools that can work with the trace file format.